content/blog/por-onde-comecar.md

```diff
@@ -1,65 +1,69 @@
 
-Vamos começar?
+Caso o Python não esteja instalado no seu computador, siga as instruções para instalação em um sistema [Windows](https://python.org.br/instalacao-windows/), [Linux](https://python.org.br/instalacao-linux/) ou [macOS](https://python.org.br/instalacao-mac/).
 
-Comece criando um programinha tipo um "Olá Mundo". Imprima uma mensagem e pronto. Veja um exemplo do programa Hello World em Python abaixo.
+Caso você não deseje instalar o Python em seu computador, recomendamos a criação de uma conta grátis no [REPL.it](https://repl.it/) onde você poderá utilizar um ambiente de edição e execuçãp de programas Python sem a necessidade de instalar software em seu computador.
+
+Vamos começar a programar?
+
+Comece criando um programinha conhecido tradicionalmente como "Olá Mundo". Imprima uma mensagem simples. Veja um exemplo do programa Hello World em Python abaixo.
 
 ~~~~
 print("Olá Mundo.")
 ~~~~
 
-Uma vez entendido o funcionament do programa Hellow World você parte para algo um pouco maior insere a segunda linha para imprimir a segunda mensagem, conforme exemplo abaixo.
+Uma vez entendido o funcionamentp do programa "Olá Mundo" você criar um programa com duas instruções conforme exemplo abaixo:
 
 ~~~~
 print("Olá Mundo.")
 print("Escreva algo aqui.")
 ~~~~
 
-Uma vez isso entendido você vai pesquisar o que é uma variável. Crie um segundo programa e atribua o valor 0 (zero) a uma variável e exiba o valor de A na tela do seu computador.
+Uma vez entendido o conceito de sequência de instruções, pesquise o que é uma variável. Crie um programa, conforme o exemplo abaixo. e atribua o valor 0 (zero) a uma variável chamada A e exiba o valor de A na tela do seu computador.
 
 ~~~~
-A=1
+A = 1
 print(A)
 ~~~~
 
-Depois pesquise como você pode adicionar 1 ao valor de A e exiba o novo valor de A na tela do seu computador. 
+Após a execução do programa anterior, pesquise como você pode adicionar 1 ao valor da variável A e exiba o novo valor de A na tela do seu computador. 
 
 ~~~~
-A=0
+A = 0
 print(A)
-A=A+1
+A = A + 1
 print(A)
 ~~~~
 
-Uma vez isso dominado insira uma segunda variável, chamada B, no seu programa e imprima o valor de A + B. 
+A seguir, insira uma segunda variável, chamada B, no seu programa e imprima o valor de A + B. 
 
 ~~~~
-A=0
+A = 0
 print(A)
-A=A+1
+A = A + 1
 print(A)
-B=35
-A=A+B
+B = 35
+A = A + B
 print(A)
 ~~~~
 
-Depois disso entendido parta para algo um pouco mais complicado e entenda como você pode usar um comando tipo "for" para repetir uma sequência de linhas do seu programa várias vezes. Sugestão: Imprima uma mensagem com o seu nome 10 vezes.
+O próximo passo seria usar o comando 'for' para repetir uma sequência de linhas do seu programa várias vezes. Sugestão: Imprima uma mensagem com o seu nome 10 vezes.
```

Comment:
Use backquotes (se usar no comentário o github interpreta) no for ao invés de single quotes, pois ele renderiza como `inline code`

Reply:
resolvidos